webapp/portlet/src/main/webapp/vue-apps/version-history-drawer/components/VersionCard.vue

```diff
@@ -0,0 +1,235 @@
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public License
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
+<template>
+  <div class="pa-0">
+    <div
+      class="author-date-wrapper d-flex justify-space-between">
+      <div class="version-author">
+        <v-chip
+          class="ma-0 px-2 font-weight-bold"
+          color="primary"
+          x-small
+          label>
+          V{{ versionObject.versionNumber }}
+        </v-chip>
+        <span class="font-weight-bold text-truncate">{{ versionObject.authorFullName }}</span>
+      </div>
+      <div class="version-update-date">
+        <date-format
+          class="text-light-color text-truncate caption"
+          :value="versionDate"
+          :format="dateTimeFormat" />
+      </div>
+    </div>
+    <div class="mb-1 description-restore-wrapper d-flex justify-space-between pt-2">
+      <div
+        v-if="enableEditDescription"
+        class="item-version-description">
+        <a
+          v-if="!versionObject.summary && descriptionInputHidden"
+          class="descriptionContent position-relative dark-grey-color"
+          @click.stop.prevent="showInput">
+          {{ $t('versionHistory.description.add') }}
+        </a>
+        <v-progress-circular
+          v-if="isUpdatingDescription"
+          :size="20"
+          color="primary"
+          indeterminate />
+        <v-tooltip
+          v-else-if="descriptionInputHidden"
+          bottom>
+          <template #activator="{ on, attrs }">
+            <p
+              v-bind="attrs"
+              v-on="on"
+              class="descriptionContent pa-0 text-truncate position-relative dark-grey-color"
+              @click.stop.prevent="showInput">
+              {{ versionObject.summary }}
+            </p>
+          </template>
+          <div
+            class="caption tooltip-version text-break">
+            {{ versionObject.summary }}
+          </div>
+        </v-tooltip>
+        <v-text-field
+          v-if="canManage"
+          ref="NewDescriptionInput"
+          v-show="!descriptionInputHidden && !isUpdatingDescription"
+          v-model="newDescription"
+          :placeholder="$t('versionHistory.description.placeholder')"
+          class="description pa-0 dark-grey-color"
+          outlined
+          dense
+          autofocus
+          @click.stop.prevent
+          @keyup.enter="updateDescription">
+          <div slot="append" class="d-flex mt-1">
+            <v-icon
+              :class="descriptionMaxLengthReached && 'not-allowed' || 'clickable'"
+              :color="descriptionMaxLengthReached && 'grey--text' || 'primary'"
+              class="px-1"
+              small
+              @click.stop.prevent="updateDescription">
+              fa-check
+            </v-icon>
+            <v-icon
```

Comment:
```suggestion
          <v-btn icon @click.stop...>
            <v-icon
```
It's always better to use `v-btn` for clickable icons for better UX with ripple and hover effects.